.github/workflows/percy.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Percy'
+on:
+  push:
+    paths:
+      - 'packages/UI/.storybook/**'
+      - 'packages/UI/src/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Node 16.x  
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Install @percy/cli and @percy/storybook
+        run: yarn add @percy/cli @percy/storybook

```

Comment:
@percy/cli and @percy/storybook are already installed in package.json why adding them in CI instead of just running yarn install?

Reply:
Its another method versus doing yarn install. Here, since this is using Percy, being more specific about what you are installing gives more information about the context.